xray/audit/jas/secretsscanner_test.go

```diff
@@ -0,0 +1,120 @@
+package jas
+
+import (
+	"errors"
+	"fmt"
+	"github.com/stretchr/testify/assert"
+	"os"
+	"path/filepath"
+	"testing"
+)
+
+func TestNewSecretsScanManager(t *testing.T) {
+	// Act
+	secretScanManager, _, err := newSecretsScanManager(&fakeServerDetails, &analyzerManagerMock{})
+
+	// Assert
+	assert.NoError(t, err)
+	assert.NotEmpty(t, secretScanManager)
+	assert.NotEmpty(t, secretScanManager.configFileName)
+	assert.NotEmpty(t, secretScanManager.resultsFileName)
+	assert.Equal(t, &fakeServerDetails, secretScanManager.serverDetails)
+}
+
+func TestSecretsScan_CreateConfigFile_VerifyFileWasCreated(t *testing.T) {
+	// Arrange
+	secretScanManager, _, _ := newSecretsScanManager(&fakeServerDetails, &analyzerManagerMock{})
```

Comment:
Let's not avoid the error returned ans assert it.
The same goes for the other executions of this API in this file.